### ISSUE TITLE
Hi, why do you setup the torch.backends.cudnn.enabled False?

### ISSUE THREAD

**Issue by wuyxiquanquan** [zzzxxxttt/pytorch_simple_CenterNet_45#1]
Disable cudnn batch normalization. Open torch/nn/functional.py and find the line with torch.batch_norm and replace the torch.backends.cudnn.enabled with False.

**Comment by zzzxxxttt** [comment]
You can refer to this [issue](https://github.com/xingyizhou/pytorch-pose-hg-3d/issues/16).

**Comment by wuyxiquanquan** [comment]
I see, thanks. But I will run seg tasks with bs = 1 and during test or eval period I still use model.train() to keep pace with the training period, but classification tasks will set bs > 128. 
If we can't ensure the bs > 128 on 1 card, sync BN is a solution(maybe other normalization also can present good acc).

**Comment by zzzxxxttt** [comment]
If you want to use syncBN, just uncomment line120 in train.py, but I have not tested training with syncBN yet.

**Comment by wuyxiquanquan** [comment]
awesome! But when did nn module implement syncBN?

**Comment by zzzxxxttt** [comment]
SyncBN is added since pytorch 1.1, you can check the [release notes](https://github.com/pytorch/pytorch/releases).

**Comment by wuyxiquanquan** [comment]
Thanks a lot! 
I checked your information just now. I'm also in USTC!!!
